### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_AzureSQLMigration_Keyboard:There is no visible indication of focus on the Data migration in progress tab.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15734]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.7;#AINotInScope;#focusvisible;#keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Data migration in progress tab item and check if there is a visible indication of focus present on the link.

### Actual:
There is no visible indication of focus on the Data migration in progress tab.
This issue is observed with Migration completed tab.

### Expected:
There should be a visible indication of focus provided for the data migration progress tab.

### User Impact:
Users using keyboard to navigate will not know where the focus is situated.

**Comment by prabhudevu** [comment]
I have verified the issue in the version 1.43 Issue is fixed.

**GithubTags:**#Closed;#Regressed:03-23-23;
